x-pack/metricbeat/module/awsfargate/task_stats/_meta/docs.asciidoc

```diff
@@ -114,14 +128,25 @@ Resources:
             - sh
             - -c
           Command:
-            - ./metricbeat setup && ./metricbeat modules disable system && ./metricbeat modules enable awsfargate && ./metricbeat -e -E cloud.id=$ELASTIC_CLOUD_ID -E cloud.auth=$ELASTIC_CLOUD_AUTH
+            - ./metricbeat setup -E cloud.id=$ELASTIC_CLOUD_ID -E cloud.auth=$ELASTIC_CLOUD_AUTH && ./metricbeat modules disable system && ./metricbeat modules enable awsfargate && ./metricbeat -e -E cloud.id=$ELASTIC_CLOUD_ID -E cloud.auth=$ELASTIC_CLOUD_AUTH
+        - Name: stress-test
```

Comment:
Maybe to add comment before this section to inform the users that this is a placeholder like container used for testing and they can replace it with their workloads:

` This is a sample container for testing purposes. Replace this with actual applications`

Reply:
Done 👍 Thanks!